homework/Vladislav Urbanski/Home work 9/UniqueWordCounter/src/main/java/com/vladis1350/bean/UniqueWordCounter.java

```diff
@@ -0,0 +1,46 @@
+package com.vladis1350.bean;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class UniqueWordCounter {
+    Map<String, Integer> uniqueWordList = new HashMap<>();
+    private int count = 1;
+
+
+    public void addWord(String word) {
+        if (uniqueWordList.containsKey(word)) {
+            uniqueWordList.put(word, uniqueWordList.get(word) + 1);
+        } else {
+            uniqueWordList.put(word, count);
+        }
+    }
+
+    public String getMostFrequentWord() {
+        int maxCount = 1;
+        for (Map.Entry<String, Integer> entryMap: uniqueWordList.entrySet()) {
+            if (entryMap.getValue() > maxCount) {
+                maxCount = entryMap.getValue();
+            }
+
+        }
+        return searchByValue(maxCount);
+    }
+
+    public String searchByValue(int value) {
+        String result = "";
+        for (Map.Entry<String, Integer> entryMap: uniqueWordList.entrySet()) {

```

Comment:
I believe Map has method to search by value

Reply:
Search for value by key is, key by value is not